hibernate-validator/build.gradle

```diff
@@ -5,13 +5,13 @@ plugins {
 dependencies {
     annotationProcessor mn.micronaut.graal
     annotationProcessor mn.micronaut.inject.java
-    compileOnly libs.glassfish.el
     compileOnly mn.micronaut.router
     implementation libs.hibernate.validator
     implementation mn.micronaut.inject
-    implementation mn.micronaut.validation
-    runtimeOnly libs.glassfish.jakarta.el
+    implementation mnValidation.micronaut.validation
 
+    testAnnotationProcessor mnValidation.micronaut.validation.processor
```

Comment:
@radovanradic why did we add a dependency to Micronaut-validation? Should not hibernate validator be independent of micronaut-validation ?

Reply:
It already had dependency to micronaut validation, this was just moving to new validation library that was moved out of core https://github.com/micronaut-projects/micronaut-hibernate-validator/pull/282/commits/978b7b24966206bb1bf01edfb40f6b60c1d4187a#diff-00f11a022501c4834933545068afa17566c38f250fc67a4f28c825987d63576dL12